debian/control

```diff
@@ -0,0 +1,13 @@
+Source: python-ev3dev
+Maintainer: Denis Demidov <dennis.demidov@gmail.com>
+Section: python
+Priority: optional
+Build-Depends: python-setuptools (>= 0.6b3), python-all-dev (>= 2.6.6-3), debhelper (>= 7)
+Standards-Version: 3.9.5
+Homepage: http://www.ev3dev.org/
```

Comment:
Since there is no homepage, I am using the git-page from now. I though it would be better to point to some ev3dev.org page, what about http://www.ev3dev.org/docs/libraries/ ?

Added Vcs*: fields.